UI polishing

matejglejtek opened the pull request:
Completion of UI according to figma design, adjustments to aircraft detail view

marianhlavac reviewed (changes requested):
The project is missing the `assets/config/uasid_prefix.json` file, cannot make the build without the file, app crashes on start. Pls include in the project if it's not sensitive.

marianhlavac commented on lib/utils/utils.dart:
```diff
@@ -158,3 +158,19 @@ Image? getFlag(String countryCode) {
+Image? getManufacturerLogo({String? manufacturer, Color color = Colors.black}) {
+  if (manufacturer == null) return null;
+  String? path;
+  if (manufacturer == 'Dronetag') path = 'assets/images/dronetag.png';
+  if (manufacturer == 'DJI') path = 'assets/images/dji_logo.png';
```
I'd later convert these logos into an icon pack (same as the DroneScannerIcons font) and use the vector variants. Just an improvement, no need to change it in this PR.

matejglejtek commented:
I swapped icons in list card for ones from design, changed spacing in card and slider headers, adjusted some colors, made flags circular etc. I will test it on different screensizes asap.

marianhlavac commented:
![Screenshot_20220826-095626](https://user-images.githubusercontent.com/3010503/186852609-35225da7-1c9a-4da2-a79a-4a42421c6da2.jpg)

There is overflow on the details and the objects on the top of the detail panel are misaligned. The icons inside the buttons are also misaligned (too large and not centered)

marianhlavac reviewed (changes requested):
The UI has been improved by this PR greatly, a lot of work went into this, however there are still a few important details missing, overflows are still present and some elements seem misaligned. I'd advise one final check against the Figma design.

Also a few conflicts has appeared, let's resolve then before we can merge.

matejglejtek commented:
Overflows were fixed, as well as position of side toolbar.

marianhlavac reviewed (changes requested):
There are still issues to be fixed, I've shared the list of issues with you.

matejglejtek commented:
I disabled landscape mode, changed layout of about dialog and showcase items. I corrected grammar is showcase texts. There is also new wifi icon without watermark.
In opendroneid library I fixed a problem that probably caused problem with starting scans.

matejglejtek commented:
I changed showcase items so they have the same styling, size and are centered to the middle of the screen.
I fixed colors of back buttons in license page and searching for places page.

marianhlavac reviewed (approved):
Alright, looks solid now, good job! Any further UI issues we'll cover in another issues.
